src/status_im2/common/scroll_page/view.cljs

```diff
@@ -145,21 +145,19 @@
                                                                     event
                                                                     "nativeEvent.contentOffset.y")))
                                             (when on-scroll
-                                              (on-scroll @scroll-height)))}
-        (when cover-image
-          [rn/view {:style {:height 151}}
-           [rn/image
-            {:source cover-image
-             ;; Using negative margin-bottom as a workaround because on Android,
-             ;; ScrollView clips its children despite setting overflow: 'visible'.
-             ;; Related issue: https://github.com/facebook/react-native/issues/31218
-             :style  {:margin-bottom -16
-                      :flex          1}}]])
-        (when children
-          [rn/view
-           {:flex             1
-            :border-radius    (diff-with-max-min @scroll-height 16 0)
-            :background-color background-color}
-           (when cover-image
-             [:f> display-picture @scroll-height cover-image])
-           children])]])))
+                                              (on-scroll @scroll-height)))
```

Comment:
is this related to mute community ? or some other changes? maybe you can explain why you made this change.

Reply:
yeah. since I am using a flat-list instead of a scroll-view these views are all placed in the header component of the flat-list. The change is only necessary because am using flat-list here